File: 프로그래머스/src/프로그래머스_02262022/자연수_뒤집어_배열로_만들기.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(long long n)
{
    vector<int> answer;
    string itos = to_string(n);

    for (int i = itos.size() - 1; i >= 0; i--)
        answer.emplace_back(itos.at(i) - '0');
    return answer;
}

File: 2026/자연수뒤집어배열로만들기.py
# https://school.programmers.co.kr/learn/courses/30/lessons/12932?language=python3
def solution(n):
    answer = []
    stack = []

    for x in str(n):
        stack.append(int(x))
    
    while stack:
        answer.append(stack.pop())
        
    return answer
